O3exp/pipeline/plotting.ini

```diff
@@ -427,7 +427,7 @@ bin-spacing = log
 psd-model = aLIGOZeroDetHighPower
 
 [page_ifar]
-decimation-factor = ${coinc-full|decimation-factor}
+decimation-factor = ${multiifo_coinc-full3d|decimation-factor}
```

Comment:
note that we might not necessarily have the same decimation factor for all coinc jobs.  
(& I wonder do we really need to give the ifar plot code the decimation factor as an option .. ie prefer it to be stored in the coinc trigger files?)

Reply:
I think the decimation factor is stored in the coinc trigger files, (was stored in the old version as well)